cymetric/convenient_interface.py

```diff
@@ -0,0 +1,213 @@
+#!/usr/bin/env python
+
+from pyne import nucname
+import pandas as pd
+import numpy as np
+import cymetric as cym
+import warnings
+
+def merge_n_drop(pdf, base_col, add_pdf, add_col):
+    """
+    Merge some additionnal columns fram an additionnal Pandas Data Frame
+    onother one and then remove the second base column (keeping SimID
+    information).
+    Parameters
+    ----------
+    pdf : Pandas Data Frame
+    base_col : list of the base columns names
+    add_pdf : Pandas Data Frame to add in the pdf one
+    add_col : columns tobe added
+    """
+    pdf = pd.merge(add_pdf[add_col], pdf, on=base_col)
+    pdf.drop(base_col[1], 1)
+    return pdf
+
+def get_reduced_pdf(pdf, rdc_list):
```

Comment:
Don't put type names in function names unless absolutely necessary.  Also, function names should be verbs.  I would suggest just `reduce()` here.